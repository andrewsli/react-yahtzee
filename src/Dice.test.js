import React from "react";
import { shallow, mount } from "enzyme";
import toJson from "enzyme-to-json";
import Dice from "./Dice";


it("renders dice without crashing", function () {
  let mockDice = [1,2,3,4,5];
  shallow(<Dice dice={mockDice} locked={false}/>);
});

it("matches snapshot", function () {
  let mockDice = [1,2,3,4,5];

  let wrapper = shallow(<Dice dice={mockDice} locked={false}/>);
  let serialized = toJson(wrapper);
  expect(serialized).toMatchSnapshot();
});