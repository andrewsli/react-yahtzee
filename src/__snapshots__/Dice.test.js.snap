// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="Dice"
>
  <Die
    idx={0}
    key="0"
    val={1}
  />
  <Die
    idx={1}
    key="1"
    val={2}
  />
  <Die
    idx={2}
    key="2"
    val={3}
  />
  <Die
    idx={3}
    key="3"
    val={4}
  />
  <Die
    idx={4}
    key="4"
    val={5}
  />
</div>
`;
